roadmap.md

```diff
@@ -39,6 +39,30 @@ gantt
 
 This section documents past RDSS sprints.  This documentation began in late July 2022, as a result of All-Hands RDSS team discussion.
 
+### 2022/08/01 - 2022/08/12
+
+```mermaid
+gantt
+    title RDSS Sprint, 2022/08/01 - 2022/08/12
+    axisFormat  %Y-%m-%dd
+    dateformat YYYY-MM-DD
+    excludes    weekends
+    section 2022 Theses
+    Import to DataSpace: th1, 2022-08-01, 10d
+    section DSpace Support
+    EZID fixes: 2022-08-11, 1d
+    section Emergency Response
+    Secrets leak fixes: 2022-08-10, 2d
+    section PDC_Describe
+    File uploading: pd1, 2022-08-01, 5d
+    DataCite implementation and ingest form: pd2, 2022-08-01, 10d
+    Globus staging deployment: 2022-08-01, 10d
```

Comment:
staging and production

Reply:
Thanks!  Added "staging and production" deployment